packages/@ngtools/webpack/src/refactor.ts

```diff
@@ -205,6 +206,9 @@ export class TypeScriptFileRefactor {
   }
 
   transpile(compilerOptions: ts.CompilerOptions): TranspileOutput {
+    if (this._fileName.endsWith('.d.ts') && this._fileName.indexOf('node_modules') > -1) {
```

Comment:
You're not verifying that a path component is `node_modules`, but just that `node_modules` is a substring. Could you use something like `this._fileName.split(path.sep).indexOf('node_modules') >= 0`?

Reply:
Thank you, you're right, I just fixed it.